app/views/anthologies/show.html.erb

```diff
@@ -48,23 +50,25 @@
           <% else %>
             <%= render 'documents_tab', anthology: @anthology %>
             <%= render 'searched_documents',
-                       search_documents_count: @search_documents_count,
-                       tab_state: @tab_state,
-                       all_documents_count: @all_documents_count,
-                       documents: @documents,
-                       anthology: @anthology
+              search_documents_count: @search_documents_count,
+              tab_state: @tab_state,
+              all_documents_count: @all_documents_count,
+              documents: @documents,
+              anthology: @anthology
             %>
-          <% end %>
-        <% else %>
+        <% end %>
+      <% else %>
+        <% if ( @anthology.user == current_user ) || current_user.admin? %>
```

Comment:
Does this prevent non-owners from seeing the documents tab, or just from adding/removing documents from the anthology?

Reply:
This prevents them from searching documents. This doesn't get rid of the document tab. document tab may be a bad naming choice.